docs/manual/common/welcome/WhereToStart.md

```diff
@@ -0,0 +1,14 @@
+# Where to begin
```

Comment:
I think the titles + page names + section names need a review. See 'Lagom Core Concepts' :

<img width="1065" alt="screen_shot_2017-02-24_at_16_53_53" src="https://cloud.githubusercontent.com/assets/762126/23310295/5cbc9d04-fab2-11e6-89fa-b75614a29cd8.png">